rules/gdi-aircraft.yaml

```diff
@@ -862,3 +862,45 @@ KODIAK:
 	WithVoxelBody:
 	WithMuzzleOverlay:
 	HitShape:
+
+Hellfire_beacon:
+	Inherits: ^AttackPlane
+	Inherits@FeatureTargettype: ^FeatureTargettypeFloraAir
```

Comment:
`FeatureTargettypeFloraAir` shouldn't exist? It definitely sounds wrong. Why does it need any TargetType at all?